app/lib/three_scale/redis_config.rb

```diff
@@ -6,6 +6,7 @@ def initialize(redis_config = {})
       raw_config = (redis_config || {}).symbolize_keys
       sentinels = raw_config.delete(:sentinels).presence
       raw_config.delete_if { |key, value| value.blank? }
+      set_pool_size(raw_config)
```

Comment:
```suggestion
      raw_config[:size] ||= raw_config[:pool_size] if raw_config.key?(:pool_size)
```

But it's fine either way.

Reply:
This wouldn't work because it doesn't remove the `:pool_size` param.

Reply:
```suggestion
      raw_config[:size] ||= raw_config.delete(:pool_size) if raw_config.key?(:pool_size)
```

Reply:
yeah, that could work